feat: add barrel blur effect (#159)
Tresjs/post-processing: docs/guide/pmndrs/barrel-blur.md

    />
    <OrbitControls auto-rotate />

    <Suspense>
      <Environment preset="shangai" />
    </Suspense>

    <RoundedBox :args="[2, 2, 2, 2, 0.25]">
      <TresMeshPhysicalMaterial
        color="white"
        :metalness=".9"
        :roughness=".5"
        :clearcoat="1.0"
        :clearcoatRoughness="0.1"
      />
    </RoundedBox>

    <Suspense>
      <EffectComposerPmndrs>
        <BarrelBlurPmndrs v-bind="effectProps" />
      </EffectComposerPmndrs>
    </Suspense>
  </TresCanvas>
</template>
```

## Props

| Prop           | Description                                                                                                                                                                  | Default                  |
| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
| **amount**     | The intensity of the barrel distortion. A value between 0 (no distortion) and 1 (maximum distortion).                                                                         | `0.1`                    |

> | **offset**     | The offset of the barrel distortion center. A `Vector2` value or an array of two numbers where both values are between 0 and 1.    | `[0.5, 0.5]`             |

Reviewer: **suggestion(non-blocking):**
```suggestion
| **offset**     | The offset of the barrel distortion center. A `Vector2` value or an A value or an array of two numbers, with both values ranging from 0 to 1.    | `[0.5, 0.5]`             |
```

Author: fixed in the last commit!
